### PR TITLE
[Chore] General Cleanup & Improvements

### PR DESCRIPTION
**Pull request by JustAHuman-xD**
I meant to open this a while ago and forgot until I saw bunkys PR (sorry bunky this supercedes that lol)

I havent tested this yet or anything but wanted to see your feedback before I do any further changes

### CODE REVIEW

**Comment by JustAHuman-xD** [comment]
and that just got merged aaaaaaaaaaaaaa

**Comment by JustAHuman-xD** [comment]
Ill fix the merge conflicts later :_)

File: src/main/java/dev/j3fftw/worldeditslimefun/commands/WorldEditSlimefunCommands.java [hunk 1]
```diff
@@ -24,7 +26,7 @@
 import org.bukkit.event.block.BlockBreakEvent;
 import org.bukkit.event.block.BlockPlaceEvent;
 
-@CommandAlias("wsef|sfedit")
+@CommandAlias("sfedit")
```
Comment (J3fftw1) [inline src/main/java/dev/j3fftw/worldeditslimefun/commands/WorldEditSlimefunCommands.java]: would like to keep the alias

File: src/main/java/dev/j3fftw/worldeditslimefun/commands/WorldEditSlimefunCommands.java [hunk 2]
```diff
@@ -34,113 +36,118 @@ public void onDefault(CommandSender sender) {
...
-            WorldEditSlimefun.getInstance().addPositionTwo(player);
-            player.sendMessage("Set position 2 to " + player.getLocation());
-        }
+    @Description("This command sets the location for position 2 for your paste/clear command.")
+    public void onPos2(Player player) {
+        WorldEditSlimefun.getInstance().addPositionTwo(player);
+        player.sendMessage("Set position 2 to " + beautifyBlockPosition(player));
+    }
+
+    public String beautifyBlockPosition(Player player) {
```
Comment (J3fftw1) [inline src/main/java/dev/j3fftw/worldeditslimefun/commands/WorldEditSlimefunCommands.java:48]: how does this return look

File: src/main/java/dev/j3fftw/worldeditslimefun/commands/WorldEditSlimefunCommands.java [hunk 3]
```diff
@@ -34,113 +36,118 @@ public void onDefault(CommandSender sender) {
...
+        final BlockPosition position = new BlockPosition(player.getLocation());
+        return "%s, %s, %s (%s)".formatted(position.getX(), position.getY(), position.getZ(), position.getWorld().getName());
     }
 
     @Subcommand("paste")
-    @CommandCompletion("@SLIMEFUN_ITEMS")
+    @Syntax("[slimefun_item]")
```
Comment (J3fftw1) [inline src/main/java/dev/j3fftw/worldeditslimefun/commands/WorldEditSlimefunCommands.java]: whats this for

**Review by J3fftw1 — CHANGES_REQUESTED** [review]

**Review by J3fftw1 — APPROVED** [review]

**Pull request merged**